File: src/main/java/org/charess/training/domain/training/PartnerTrainingParticipants.java
package org.charess.training.domain.training;

import org.charess.training.domain.security.Person;
import org.charess.training.domain.security.Place;

import java.util.List;

public class PartnerTrainingParticipants {

    private Training training;
    private Place partner;
    private List<Person> participants;

    public Training getTraining() {
        return training;
    }

    public void setTraining(Training training) {
        this.training = training;
    }

    public Place getPartner() {
        return partner;
    }

    public void setPartner(Place partner) {
        this.partner = partner;
    }

    public List<Person> getParticipants() {
        return participants;
    }

    public void setParticipants(List<Person> participants) {
        this.participants = participants;
    }
}

File: src/main/webapp/src/app/core/models/training/partner.training.participants.ts
import {Institution} from "../security/institution";
import {Training} from "./training";

export class PartnerTrainingParticipants {
    id: number;
    training: Training;
    partner: Institution;
    participants: [{Person}];

    constructor(t) {
        {
            this.id = t.id;
            this.training = t.training;
            this.partner = t.partner;
            this.participants = t.participants;
        }
    }
}

